tests/providers/cncf/kubernetes/operators/test_kubernetes_pod.py

```diff
@@ -893,6 +894,7 @@ def test_mark_checked_unexpected_exception(self, mock_patch_already_checked, moc
     @mock.patch(f"{POD_MANAGER_CLASS}.await_xcom_sidecar_container_start")
     def test_wait_for_xcom_sidecar_iff_push_xcom(self, mock_await, mock_extract_xcom, do_xcom_push):
         """Assert we wait for xcom sidecar container if and only if we push xcom."""
+        self.hook_mock.return_value.get_xcom_sidecar_container_image.return_value = "alpine"
```

Comment:
seems you should not need this change?

Reply:
@dstandish without this, the following error is raised:
`yaml.representer.RepresenterError: ('cannot represent an object', <MagicMock name='KubernetesHook().get_xcom_sidecar_container_image().to_dict()' id='139908045486608'>)`